zebra-chain/src/work/equihash.rs

```diff
@@ -61,6 +61,13 @@ impl Solution {
     }
 }
 
+#[cfg(feature = "getblocktemplate-rpcs")]
+impl Default for Solution {
+    fn default() -> Self {
+        Self([0; SOLUTION_SIZE])
+    }
+}
```

Comment:
This default doesn't obey the consensus rules. So usually I would:
- make a named method that makes it clear it's only for a proposal
- make it test-only

Reply:
Moved/renamed to `for_proposal` method in [`516f17c`](https://github.com/ZcashFoundation/zebra/pull/5884/commits/516f17c257e3f39f63a9fa91ec0f0d253ca02b99).